discretize/TensorMesh.py

```diff
@@ -159,6 +159,22 @@ def gridN(self):
         """Nodal grid."""
         return self._getTensorGrid('N')
 
+    @property
+    def h_gridded(self):
+        """Dimensions of all cells in order."""
```

Comment:
would you mind including dimensions in the docstring? eg
```
returns an nC x dim array of the cell dimensions (eg. [hx, hy, hz] for a 3D mesh) 
```

Reply:
Oh i see. Once of those